src/java/picard/illumina/ExtractIlluminaBarcodes.java

```diff
@@ -73,19 +73,49 @@
  * @author jburke@broadinstitute.org
  */
 @CommandLineProgramProperties(
-        usage = "Determine the sample barcode for each read in an Illumina lane.\n" +
-                "For each tile, a file is written to the basecalls directory of the form s_<lane>_<tile>_barcode.txt. " +
-                "An output file contains a line for each read in the tile, aligned with the regular basecall output. \n" +
-                "The output file contains the following tab-separated columns: \n" +
-                "    * read subsequence at barcode position\n" +
-                "    * Y or N indicating if there was a barcode match\n" +
-                "    * matched barcode sequence\n" +
-                "Note 1: that the order of specification of barcodes can cause arbitrary differences in output for poorly matching barcodes.\n" +
-                "Note 2: molecular barcodes (M in the read structure) are not the barcode being extracted here and will be ignored here.\n\n",
-        usageShort = "Tool to determine the barcode for each read in an Illumina lane",
+
+        usage = ExtractIlluminaBarcodes.USAGE_SUMMARY + ExtractIlluminaBarcodes.USAGE_DETAILS,
+        usageShort = ExtractIlluminaBarcodes.USAGE_SUMMARY,
         programGroup = Illumina.class
 )
 public class ExtractIlluminaBarcodes extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Tool to determine the barcode for each read in an Illumina lane.  ";
```

Comment:
"Tool to determine the barcode" -> "Determines the barcode" to be consistent with the other tool summaries.